force-app/main/default/classes/DMLOperations.cls

```diff
@@ -96,13 +97,13 @@ public with sharing class DMLOperations {
      */
     public static void updateOpportunityStage(Id oppId, String newStage) {
         // Retrieve the Opportunity from the database
+        Opportunity myOpp = [SELECT Id FROM Opportunity WHERE Id = :oppId];
         // [SELECT Id FROM Opportunity WHERE Id = :oppId]
```

Comment:
It's worth deleting some of these comments — for instance, these SOQL queries — since they don't serve any purpose now that your code has been written. They just clutter up your method.